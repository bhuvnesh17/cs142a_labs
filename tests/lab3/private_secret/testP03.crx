// print case sensitive vars

var a : int;

func main() : void {
  var laMbda : float;
  var lambda : int;

  ::unk(a);
}