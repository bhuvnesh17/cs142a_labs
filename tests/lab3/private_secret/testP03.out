Error parsing file.
ResolveSymbolError(9,5)[Could not find unk.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(a)
Symbol(main)
  Symbol(laMbda)
  Symbol(lambda)


